src/main/java/org/jabref/JabRefException.java

```diff
@@ -3,11 +3,14 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+// Class for handling exceptions in GUI and printing corresponding exception messages
 public class JabRefException extends Exception {
-
+    
+    // Creating a logger and using it to write messages to log files
```

Comment:
Not necessary as naming of variable is already self-explanatory. "write to logfiles" depends on log4j configuration that is maintained elsewhere - thus: obsolete and misleading comment here.

Reply:
I personally faced a problem of googling log4j. For the sole purpose I wrote this because it is not common for students to use logger in their JavaFx project thus they do not know about this.